src/main.c

```diff
@@ -56,10 +56,18 @@ print_usage()
            "                (used as generation id for postgres)\n"
            "-f | -F : consumer / producer filename (only file)\n"
            "-l      : path to the log file\n"
+           "-V      : print version\n"
            "\n");
     exit(1);
 }
 
+void
+print_version()
+{
+    printf("schaufel version: %s\n\n", _SCHAUFEL_VERSION);
```

Comment:
The interpolation is not necessary (might as well write `"schaufel version: "_SCHAUFEL_VERSION"\n"`)

Reply:
Good catch.